pvlib/iotools/tmy.py

```diff
@@ -62,80 +59,85 @@ def read_tmy3(filename, coerce_year=None, recolumn=True):
-    TMYData.WspdUncertainty             See [1]_, Table 1-6
-    TMYData.Hvis                        Distance to discernable remote objects at time indicated (7777=unlimited), meter
-    TMYData.HvisSource                  See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.HvisUncertainty             See [1]_, Table 1-6
-    TMYData.CeilHgt                     Height of cloud base above local terrain (7777=unlimited), meter
-    TMYData.CeilHgtSource               See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.CeilHgtUncertainty          See [1]_, Table 1-6
-    TMYData.Pwat                        Total precipitable water contained in a column of unit cross section from earth to top of atmosphere, cm
-    TMYData.PwatSource                  See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.PwatUncertainty             See [1]_, Table 1-6
-    TMYData.AOD                         The broadband aerosol optical depth per unit of air mass due to extinction by aerosol component of atmosphere, unitless
-    TMYData.AODSource                   See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.AODUncertainty              See [1]_, Table 1-6
-    TMYData.Alb                         The ratio of reflected solar irradiance to global horizontal irradiance, unitless
-    TMYData.AlbSource                   See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.AlbUncertainty              See [1]_, Table 1-6
-    TMYData.Lprecipdepth                The amount of liquid precipitation observed at indicated time for the period indicated in the liquid precipitation quantity field, millimeter
-    TMYData.Lprecipquantity             The period of accumulatitudeion for the liquid precipitation depth field, hour
-    TMYData.LprecipSource               See [1]_, Table 1-5, 8760x1 cell array of strings
-    TMYData.LprecipUncertainty          See [1]_, Table 1-6
-    TMYData.PresWth                     Present weather code, see [2]_.
-    TMYData.PresWthSource               Present weather code source, see [2]_.
-    TMYData.PresWthUncertainty          Present weather code uncertainty, see [2]_.
-    =============================       ======================================================================================================================================================
+    =====================       ======================================================================================================================================================
+    field                       description
+    =====================       ======================================================================================================================================================
+    Index                       A pandas datetime index. NOTE, the index is timezone aware, and times are set to local time (daylight savings is not included)
```

Comment:
I would keep this as "local standard time".  "local time" alone implies DST *is* included, for most places in the US anyway.